clients/java/src/main/java/io/zeebe/client/impl/ZeebeClientBuilderImpl.java

```diff
@@ -44,6 +44,7 @@
   private String brokerContactPoint = "0.0.0.0:26500";
   private int jobWorkerMaxJobsActive = 32;
   private int numJobWorkerExecutionThreads = 1;
+  private float defaultJobWorkerMinJobsActiveRatio = 0.3f;
```

Comment:
I tried to follow the naming convention of the newer properties. The names of the older ones are obviously historically grown but not confusing enough to make a big drama out of it. Maybe JavaDocs could be double-checked for clarity.

Reply:
@korthout should we quickly take the chance and fix these historically grown property names for 1.0.0?

Reply:
@npepinpe I feel the breaking changes I proposed here come too late for 1.0, but I'd like to know what you think.

Reply:
We can do them post 1.0 and live with having the previous ones deprecated. I think post 1.0 it would be good if product took some time to do some user research to figure out how the client API should look like - in the mean time, I wouldn't put too much effort here.